packages/core/src/input/Input.css

```diff
@@ -1,6 +1,6 @@
 /* Style applied to the root element */
 .saltInput {
-  --input-borderColor: var(--salt-editable-borderColor);
+  --input-borderColor: var(--saltInput-borderColor, var(--salt-editable-borderColor));
```

Comment:
and then the 1px solid would also have to be redefined
unless we want to just split it into border-bottom-color, border-bottom-style etc and have then only the color one exposed
I would ask design for their input on which way we want to go (who thought this would be so complicated lol 😄 )